main.tf

```diff
@@ -4,6 +4,17 @@ provider "aws" {
   region                  = var.region
 }
 
+terraform {
+  backend "remote" {
+    hostname     = "app.terraform.io"
+    organization = "CovidZero"
+
+    workspaces {
+      name = var.workspace
```

Comment:
@thiagolsfortunato interpolation is not possible on Terraform backend, you must pass the string name of your workspace.

![image](https://user-images.githubusercontent.com/3285936/77274688-3f879b00-6c95-11ea-8d36-ee1d00f4d307.png)